docs/source/using_the_ve/virtual_ecosystem_in_use.md

```diff
@@ -89,15 +89,16 @@ for file in example_files:
 ## Run the Virtual Ecosystem model
 
 Now the example data and configuration have been set up, the `ve_run` command can be
-used to execute a Virtual Ecosystem simulation. The `progress` option shows the progress
-of the simulation through the various modelling stages.
+used to execute a Virtual Ecosystem simulation. When the detailed logging is redirected
+to a file, the command generates a short progress report to show the model running. This
+can be made shorter or completely muted by using the `-q` argument: repeat the argument
+to remove more details (e.g. `-qq` or `-qqq`).
 
 ```{code-cell} ipython3
 %%bash
 ve_run /tmp/ve_example/config \
     --out /tmp/ve_example/out \
-    --logfile /tmp/ve_example/out/logfile.log \
-    --progress \
+    --logfile /tmp/ve_example/out/logfile.log
```

Comment:
This reminds me [something we did in another project](https://github.com/EnergySystemsModellingLab/MUSE_OS/blob/97da23fe3eecce7d03b0f93e4d0e052897d55ef0/src/muse/__init__.py#L42) - also done for jobs in the HPC - where we separate the logs in two files: in one we show the normal log information showing progress (INFO or below), while the other shows important information that a user must pay attention (WARNING and above). This way, warnings are not lost in the mess of other messages and are more easily spotted by users. Just a suggestion for the future, not something to change in this PR.

Reply:
Agreed 💯 . In the unlikely use case that a run is logging to the console, you would see the whole thing, but this mini report does not currently show anything if it breaks. You'd have to go the log.